hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/ConversionUtils.java

```diff
@@ -636,6 +650,24 @@ public static byte[] asEvmAddress(final long num) {
         return copyToLeftPaddedByteArray(num, new byte[20]);
     }
 
+    /**
+     * Given a long entity number, returns its 20-byte EVM address.
+     *
+     * @param shard the shard number
+     * @param realm the realm number
+     * @param num the entity number
+     * @return its 20-byte EVM address
+     */
+    public static byte[] asEvmAddress(final long shard, final long realm, final long num) {
```

Comment:
Missing unit test (though it certainly seems as though the code is obvious, but of course unit tests are also for _maintenance_ purposes, i.e., future changes).